Documentation/Articles/querying_for_data.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Querying for Data
+description: About Reading data
+keywords: read model, query, ddd, read, cqrs,  
+author: pavsaund
+---
+
+
+# Querying for Data
+
+## Introduction
+The query or read side of the CQRS pattern is all about exposing the current state of a system in a format that is ready to be consumed by interaction layers (ie: web-based user interface). The state should be exposed through models that are optimized for querying, and is usually updated by changes made to a system through events.
+
+{{% notice note %}}
+Dolittle exposes read data through two different mechanisms:
+ 
+ - [Read Models](https://github.com/dolittle/Runtime/tree/master/Documentation/read/read_model.md)
+ - [Queries](https://github.com/dolittle/Runtime/tree/master/Documentation/read/query.md)
+{{% /notice %}}
+
+### Updating Read Data
+[Event Processors](LINK TO EVENT PROCESSORS) are responsible for consuming events in the system and updating read-models.
```

Comment:
I would say "subscribing to events" and updating read-models when state changes.

Reply:
The intention here is to link a reader to the "previous" step of actually generating the data from events. I can remove it to keep the document more focused